libs/platform/src/lib/components/form/checkbox/checkbox.component.ts

```diff
@@ -35,6 +35,39 @@ let nextUniqueId = 0;
     providers: [{ provide: FormFieldControl, useExisting: CheckboxComponent, multi: true }]
 })
 export class CheckboxComponent extends BaseInput implements AfterViewInit {
+    /**
+     * Attached to the aria-label attribute of the host element. In most cases, aria-labelledby will
+     * take precedence so this may be omitted.
+     */
+    @Input('aria-label')
+    ariaLabel = '';
```

Comment:
aria-label, aria-labelled by,aria-disabled is widely used along most of the components, Won't it help to have them in base class

Reply:
@fkolar @KevinOkamoto , what do you suggest.

Reply:
I don't think an input is necessary, on core, application developer can simply use <fd-checkbox aria-label="some label">, so maybe the same thing here?

Reply:
tried without Input aria-label by passing aria-label/attr.aria-label to fdp-checkbox, but it does not work.
Need to paas aria-label to native html element, so the screen reader can work correctly.

Reply:
I agree we probably should put `ariaLabel`, `ariaLabelledBy`, etc. in the BaseInput class. However, I think we should do that in a separate PR. This will effect a lot of components, and I don't think it's appropriate to add to the file changes in this PR.

Let's create a GitHub issue for the `ariaLabel` in BaseInput, and leave this code the way it is.

Reply:
I was about to ask dont we have these on BaseInput?

Reply:
@mikerodonnell89 you need that in core as you have wrapping component

Reply:
Ok that makes sense... needs to be on the input element specifically